demos/storybook/stories/dropdown-toolbar/with-full-config.stories.ts

```diff
@@ -0,0 +1,49 @@
+import { text, number, select } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+
+export const withFullConfig = (): any => ({
+    styles: [
+        `
+            [dir='rtl'] .settings-icon {
+                margin-right: 0;
+                margin-left: -8px;
+            }
+
+            .settings-icon {
+                margin-right: -8px;
+            }
+        `,
+    ],
+    template: `
+       <pxb-dropdown-toolbar [title]="title" [subtitle]="state.selected || subtitle">
+            <button mat-icon-button pxb-nav-icon (click)="clickPXBIcon()" aria-label="menu icon">
+            <mat-icon *ngIf="navIcon === 'menu'">menu</mat-icon>
+            <mat-icon *ngIf="navIcon === 'arrow_back'">arrow_back</mat-icon>
+            </button>
+        <ng-container pxb-toolbar-menu>
+            <button mat-menu-item (click)="updateSubtitle('Test Item 1', state)">Test Item 1</button>
+            <button mat-menu-item (click)="updateSubtitle('Test Item 2', state)">Test Item 2</button>
+            <button mat-menu-item (click)="updateSubtitle('Test Item 3', state)">Test Item 3</button>
+        </ng-container>
+        <div>
+            <button mat-icon-button *ngIf="count > 0" (click)="clickRightContentIcon()" aria-label="home icon"><mat-icon>home</mat-icon></button>
```

Comment:
The spacing on these icons is still off from what we recommend (24px between icons, edge to edge, and 16px between rightmost icon edge and toolbar end)